src/errors.rs

```diff
@@ -33,4 +33,17 @@ impl From<TcnError> for ServicesError {
   }
 }
 
+impl From<String> for ServicesError {
+  fn from(error: String) -> Self {
+    ServicesError::Error(Box::new(StdError::new(ErrorKind::Other, error)))
```

Comment:
@i-schuetz  What is the reason for Box-ing the error?

Reply:
The content of the box (`dyn std::error::Error + Send + Sync + 'static`) are traits, so the compiler can't know the actual types/sizes. `Box` is a pointer so it has a fixed size. Most places (like e.g. `ServicesError` where `Error` is used) require a sized type.